modules/data-entry/src/main/frontend/src/questionnaireEditor/AnswerOptions.jsx

```diff
@@ -28,77 +28,68 @@ import {
-  let [ newUUID, setNewUUID ] = useState([]);
-  let [ newValue, setNewValue ] = useState([]);
   let [ deletedOptions, setDeletedOptions ] = useState([]);
-  let [ tempValue, setTempValue ] = useState(''); // Holds new, non-comitted answer options
+  let [ tempValue, setTempValue ] = useState(''); // Holds new, non-committed answer options
+  let [ isDuplicate, setIsDuplicate ] = useState(false);
 
   // Clear local state when data changes
   useEffect(() => {
     setOptions(Object.values(data).filter(value => value['jcr:primaryType'] == 'lfs:AnswerOption').slice());
-    setNewUUID([]);
-    setNewValue([]);
     setDeletedOptions([]);
     setTempValue('');
+    setIsDuplicate(false);
   }, [data])
 
-  let deleteOption = (value) => {
-    setOptions(oldOptions => {
-      let newOptions = oldOptions.slice();
-      newOptions.splice(newOptions.indexOf(value), 1);
-      return newOptions;
-    })
-
+  let deleteOption = (index) => {
     setDeletedOptions(old => {
       let newDeletedOptions = old.slice();
-      newDeletedOptions.push(value);
+      newDeletedOptions.push(options[index]);
       return newDeletedOptions;
-    })
-  }
+    });
 
-  let updateInsertedOption = (index, event) => {
-    setNewValue(oldValue => {
-      var value = oldValue.slice();
-      value[index] = event.target.value;
-      return value;
-    })
+    setOptions(oldOptions => {
+      let newOptions = oldOptions.slice();
+      newOptions.splice(index, 1);
+      return newOptions;
+    });
   }
 
-  let deleteInsertedOption = (index, event) => {
-    setNewValue(oldValue => {
-      let value = oldValue.slice();
-      value.splice(index, 1);
-      return value;
-    })
-    setNewUUID(oldUuid => {
-      let uuid = oldUuid.slice();
-      uuid.splice(index, 1);
-      return uuid;
-    });
+  let validateOption = (optionInput) => {
+    if (optionInput) {
+      setIsDuplicate(false);
+      let inputs = (optionInput + ' ').split(/=(.+)/);
+      let duplicateOption = options.find( option => option.value.trim() === inputs[0].trim() || inputs[1] && (option.label?.trim() === inputs[1].trim()));
+      duplicateOption && setIsDuplicate(true);
+    }
   }
 
-  let handleInputOption = (event) => {
-    if (tempValue && !newValue.includes(tempValue)) {
-      setNewUUID(oldUuid => {
-        let tempUUID = oldUuid.slice();
-        tempUUID.push(uuidv4());
-        return tempUUID;
-      });
+  let handleInputOption = (optionInput) => {
+    if (optionInput && !isDuplicate) {
+      // The text entered on each line should be split
+      // by the first occurrence of the separator = if the separator exists
+      // e.g. F=Female as <value> = <label>
+      let inputs = optionInput.split(/=(.+)/);
+      let newOption = {};
+      newOption.value = inputs[0].trim();
+      let sanitisedValue = newOption.value.trim().replace(/\s/g, '_').replace(/\W/g, '');
+      newOption["@path"] = path + "/" + sanitisedValue + stringToHash(newOption.value);
```

Comment:
Bug: options whose values that start with numbers will still be displayed at the beginning (thank you @veronikaslc for explaining how nodes are grouped, `@name`s starting with numbers are separated from `@name`s starting with letters).
Suggestion: we can remove `sanitizedValue` completely and prefix the `@name` of each option with the same string, e.g:
```suggestion
      newOption["@path"] = path + "/AnswerOption" + stringToHash(newOption.value);
```